cypress/integration/mocked/onboarding_flow.spec.ts

```diff
@@ -191,17 +195,12 @@ describe('Onboarding flow', () => {
       CommunicationsPage.backButton().should('not.exist');
 
       CommunicationsPage.allCheckboxes().should('not.be.checked');
-      CommunicationsPage.allOptoutCheckboxes()
-        // select parent (to avoid cypress element not visible error)
-        .parent()
-        .click({ multiple: true });
 
       // mock form save success
       cy.mockNext(200);
 
       CommunicationsPage.saveAndContinueButton().click();
-
-      cy.lastPayloadIs([{ id: 'supporter', consented: true }]);
+      cy.lastPayloadIs([{ id: 'supporter', consented: false }]);
```

Comment:
I believe these tests going through the full flow were actually testing the wrong things? The one opting in to everything opted out of the supporter consent, and the one opting out of everything opted in. So I've adjusted things here.